src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt

```diff
@@ -56,27 +60,17 @@ class ExternalDocRequestEndpointIntTest : IntegrationTestBase() {
     @BeforeEach
     fun beforeEach() {
         courtCaseEventsQueue?.sqsClient?.purgeQueue(PurgeQueueRequest.builder().queueUrl(courtCaseEventsQueue!!.queueUrl).build())
-        amazonS3.createBucket(bucketName, "eu-west-2")
+        amazonS3.createBucket(CreateBucketRequest.builder().bucket(bucketName).build())
```

Comment:
Does it matter that the region "eu-west-2" is not set  as it looks like it is gone?

Reply:
It is set by default: 
https://github.com/ministryofjustice/crime-portal-gateway/blob/PIC-4153-upgrade-s3/src/test/resources/application-test.yml#L28

Reply:
<img width="744" alt="image" src="https://github.com/user-attachments/assets/ae244f65-0ba6-49fd-95a8-1ea88c6b0762">

Reply:
See this debug breakpoint showing the region set on the ObjectListing